plugin/log_assert.py

```diff
@@ -177,7 +185,16 @@ def pytest_runtest_makereport(item, call):
 def get_logger(*args, **kwargs):
     if 'verbose_lvl' not in kwargs:
         kwargs['verbose_lvl'] = 0
-    return mrglog.get_logger(*args, **kwargs)
+    if 'output' not in kwargs:
+        kwargs['output'] = "std, txt"
```

Comment:
It doesn't have to be, if someone decides that he/she doesn't want std output from logger, than he/she could. We don't require it, but we want that as default.